README.md

```diff
@@ -140,3 +140,28 @@ secrets.
           twitter_consumer_secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
           twitter_consumer_key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
 ```
+
+### Deploy to Mastodon
+
+To deploy to Mastodon (in addiction to Slack) you are required to set `deploy_mastodon`
+to true, and also define all the needed environment variables in your repository
+secrets.
+
+```yaml
+      - id: updater
+        name: Job Updater
+        uses: rseng/jobs-updater@add/deploy-arg
```

Comment:
We will probably want this version to be "main" for the docs so the user chooses the latest.
```suggestion
        uses: rseng/jobs-updater@main
```

Reply:
Yeah, good call.  To be honest, I wasn't even convinced that branch was still available.